website/templates/emails/security_permissions_change.txt.mako

```diff
@@ -0,0 +1,25 @@
+Dear ${user.given_name},
+
+We are writing to let you know of a change to our permission model that may affect how you choose to organize your projects.
```

Comment:
"We are writing to let you know of a change to our permission model that may affect how you choose to organize your projects on the Open Science Framework."